### ISSUE TITLE
Installation error: cannot stat 'prrte-default-hostfile'

### ISSUE THREAD

**Issue by tonycurtis** [openpmix/prrte#366]
Thank you for taking the time to submit an issue!

## Background information

Install fails due to missing file

### What version of the PMIx Reference Server are you using? (e.g., v1.0, v2.1, git master @ hash, etc.)

github master @ 2d99acbb4aa6fe1be19c23320f47a983851ad36e

##### What version of PMIx are you using? (e.g., v1.2.5, v2.0.3, v2.1.0, git branch name and hash, etc.)

github master @ d2473b0e641709bb8395823d07d39726543486ae

### Please describe the system on which you are running

* Operating system/version: Fedora 30
* Computer hardware: x86_64
* Network type: eth (mainly shared memory)

-----------------------------

## Details of the problem

```shell
$ make install
...
...
...
make[3]: Entering directory '/home/arcurtis/src/prrte/build/src/etc'
make[3]: Nothing to be done for 'install-exec-am'.
/usr/bin/mkdir -p /home/arcurtis/opt/prrte/git/etc
******************************* WARNING ************************************
*** Not installing new prrte-mca-params.conf over existing file in:
***   /home/arcurtis/opt/prrte/git/etc/prrte-mca-params.conf
******************************* WARNING ************************************
/usr/bin/install: cannot stat 'prrte-default-hostfile': No such file or directory
make[3]: *** [Makefile:866: install-data-local] Error 1
make[3]: Leaving directory '/home/arcurtis/src/prrte/build/src/etc'
make[2]: *** [Makefile:749: install-am] Error 2
make[2]: Leaving directory '/home/arcurtis/src/prrte/build/src/etc'
make[1]: *** [Makefile:1862: install-recursive] Error 1
make[1]: Leaving directory '/home/arcurtis/src/prrte/build/src'
make: *** [Makefile:843: install-recursive] Error 1
```

**Comment by rhc54** [comment]
I take it you are doing a VPATH build? If so, I think that should be fixed in #367

**Comment by tonycurtis** [comment]
Yep, looks good
